examples/hacker_news_pwa/web/index.html

```diff
@@ -5,6 +5,7 @@
   <meta name="theme-color" content="#212121">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta name="Description" content="A Hacker News reader implemented as a Progressive Web App using AngularDart.">
+  <link rel="prefetch" href="https://node-hnapi.herokuapp.com/news?page=1" as="fetch" crossorigin />
```

Comment:
Why "prefetch" instead of "preload?

Reply:
Preload isn't appropriate unless its 100% going to hit that URL.

(Chrome Dev Console warns you as such)